content/integrations/google_cloud_platform.md

```diff
@@ -3,6 +3,7 @@ title: Datadog-Google Cloud Platform Integration
 integration_title: Google Cloud Platform
 kind: integration
 git_integration_title: google_cloud_platform
+description: "Collect a wealth of GCP metrics and visualize your instances in a host map."
```

Comment:
I'm not sure "wealth of" is appropriate here. Either we should describe the types of available metrics or explain why it is useful to have them.  GCP should probably be spelled out.